src/styles/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  width: ($cell-size * 7) + 6px;
```

Comment:
where did you get `6px`?

Reply:
6 days, every day with 1px gap, I must rewrite like 6 * $gap and give for gap variable?